### PR TITLE
Use node16 as the extension runtime

### PR DESCRIPTION
**Pull request by bidoubiwa**
Fixes: https://github.com/meilisearch/integration-guides/issues/196

### CODE REVIEW

File: functions/package.json [hunk 1]
```diff
@@ -16,9 +17,6 @@
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage"
   },
-  "engines": {
```
Comment (brunoocasali) [inline functions/package.json:19]: Why do you want to remove this protection?

Comment (bidoubiwa) [reply functions/package.json:19]: I took inspiration of another repo pour you are right! I'm putting it back :)

Comment (bidoubiwa) [reply functions/package.json:19]: As per: https://github.com/algolia/firestore-algolia-search/blob/main/functions/package.json I added a limit on node v15 and plus

**Review by brunoocasali — APPROVED** [review]
LGTM!

**Comment by bidoubiwa** [comment]
bors merge

**Comment by brunoocasali** [comment]
bors merge

**Comment by bidoubiwa** [comment]
bors merge

**Comment by bidoubiwa** [comment]
bors try

**Comment by bidoubiwa** [comment]
Closed in favor of new PR

**Pull request closed without merging**